src/Features/CSharp/Portable/Completion/KeywordRecommenders/NullKeywordRecommender.cs

```diff
@@ -14,37 +12,9 @@ public NullKeywordRecommender()
         {
         }
 
-        protected override bool IsValidContext(int position, CSharpSyntaxContext context, CancellationToken cancellationToken)
-        {
-            var syntaxTree = context.SyntaxTree;
-
-            return
-                context.IsAnyExpressionContext ||
-                context.IsStatementContext ||
-                context.IsGlobalStatementContext ||
-                IsInSelectCaseContext(context, cancellationToken);
-        }
-
-        private bool IsInSelectCaseContext(CSharpSyntaxContext context, CancellationToken cancellationToken)
-        {
-            var token = context.TargetToken;
-            if (token.Kind() != SyntaxKind.CaseKeyword)
-            {
-                return false;
-            }
-
-            var switchStatement = token.GetAncestor<SwitchStatementSyntax>();
-            if (switchStatement != null)
-            {
-                var info = context.SemanticModel.GetTypeInfo(switchStatement.Expression, cancellationToken);
-                if (info.Type != null &&
-                    info.Type.IsValueType)
-                {
-                    return false;
-                }
-            }
-
-            return true;
-        }
+        protected override bool IsValidContext(int position, CSharpSyntaxContext context, CancellationToken cancellationToken) =>
```

Comment:
nit: in the IDE we put the => on the next line for methods

Reply:
got it! i was going to do that until i noticed that it's consistent in code all around to place operators at the end of the line, see `||` below - if I was going to put `=>` on the next line, I would like to do the same with `||` as well

Reply:
or my question is rather: how do you then indent the expression if it's multiline? I would rather not have some non-standard indentation such as 6 characters on every following line just because of the `=>` on the first line, like this:
```c#
    => context.IsAnyExpressionContext ||
       context.IsStatementContext ||
       context.IsGlobalStatementContext;
```
the other option of losing alignment is unappealing as well:
```c#
    => context.IsAnyExpressionContext ||
    context.IsStatementContext ||
    context.IsGlobalStatementContext;
```

Reply:
We do the former.  

> I would rather not have some non-standard indentation

We distinguish indentation with alignment.  The code is "indented" 4 from the parent.  But the expression components are aligned.

Reply:
ok, I can do that, but I think the intent of the way the original code was written - by having `return` on a separate line, was exactly to avoid that :smiley: